content/documentation/pages/1-installation/3-kubernetes/1-helm.md

```diff
@@ -556,7 +556,17 @@ do so:
 ```yaml
 env:
   - name: SPRING_APPLICATION_JSON
-    value: '{ "maven": { "local-repository": null, "remote-repositories": { "repo1": { "url": "https://repo.spring.io/libs-snapshot"} } } }'
+    value: -|
```

Comment:
I think it should be: `value: |-` instead.

Reply:
Hmmm, that's what's in my local copy

Reply:
I see. I was testing all my manifests with `value: |-`, so I wasn't sure whether what you have here would work. Did you try with `value: -|`, and it works?